backend/lcfs/web/api/fuel_code/repo.py

```diff
@@ -960,7 +971,7 @@ async def get_target_carbon_intensity(
         )
         result = await self.db.execute(stmt)
 
-        return result.scalar_one()
+        return result.unique().scalar_one()
```

Comment:
Still here

Reply:
We are adding compliance period here besides TargetCarbonIntensity. We are required to use unique().